src/test/scala/org/nbastats/scraper/ScraperSpec.scala

```diff
@@ -17,5 +17,14 @@ class ScraperSpec extends BaseSpec {
     timmy.pointsPerGame should be (19.0)
     timmy.reboundsPerGame should be (10.8)
     timmy.assistsPerGame should be (3.0)
+  }
+
+  @Test
+  def jrue(): Unit = {
+    val jrue: GameStat = Scraper.scrapeGameStat("https://www.basketball-reference.com/players/h/holidjr01/gamelog/2010", 1)
+
+
```

Comment:
nice, the last thing we need to do here is actually assert something about jrue.

`jrue.points should be (2)` or something like that

Reply:
filled it in bruv